swift-collections.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>FILEHEADER</key>
+    <string>===----------------------------------------------------------------------===//
```

Comment:
```suggestion
<string>//===----------------------------------------------------------------------===//
```

I think this line is missing a "//"

Reply:
I thought the same thing, but it looks like Xcode hardcodes inserting the leading the `//`

Reply:
You're right, Xcode does insert the leading "//". I should've checked it first before commenting. 

Weirdly though, this template doesn't for me locally, and it only seem to work in an Xcode project.